packages/google_maps_flutter/google_maps_flutter_web/example/integration_test/marker_test.dart

```diff
@@ -205,5 +206,21 @@ void main() {
         }, throwsAssertionError);
       });
     });
+
+    testWidgets('interpretCorrectZIndexInConversion',
```

Comment:
I didn't find any conversion tests in the web package, so Im not 100% sure if this is in the right place. Maybe it should be in a dedicated convert_test file